src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj

```diff
@@ -8,7 +8,7 @@
 		<Description>Provides management capabilities for Microsoft Azure IotHub.</Description>
 		<AssemblyTitle>Microsoft Azure IotHub Management</AssemblyTitle>
 		<AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-		<Version>1.1.3</Version>
+		<Version>1.1.4</Version>
```

Comment:
Looks like you might be introducing some breaking changes here, @shahabhijeet can you please verify,
This might need a major version change.